Source/Core/Core/FreeLookManager.cpp

```diff
@@ -0,0 +1,237 @@
+
+  m_other_buttons->AddInput(ControllerEmu::Translate, _trans("Reset View"));
+
+  groups.emplace_back(m_fov_buttons = new ControllerEmu::Buttons(_trans("Field of View")));
+
+  m_fov_buttons->AddInput(ControllerEmu::Translate, _trans("Increase X"));
+  m_fov_buttons->AddInput(ControllerEmu::Translate, _trans("Decrease X"));
+  m_fov_buttons->AddInput(ControllerEmu::Translate, _trans("Increase Y"));
+  m_fov_buttons->AddInput(ControllerEmu::Translate, _trans("Decrease Y"));
+}
+
+std::string FreeLookController::GetName() const
+{
+  return std::string("FreeLook") + char('1' + m_index);
+}
+
+void FreeLookController::LoadDefaults(const ControllerInterface& ciface)
+{
+  EmulatedController::LoadDefaults(ciface);
+
+  auto hotkey_string = [](std::vector<std::string> inputs) {
+    return "@(" + JoinStrings(inputs, "+") + ')';
+  };
+
+  m_move_buttons->SetControlExpression(MoveButtons::Up, hotkey_string({"Shift", "E"}));
+  m_move_buttons->SetControlExpression(MoveButtons::Down, hotkey_string({"Shift", "Q"}));
+  m_move_buttons->SetControlExpression(MoveButtons::Left, hotkey_string({"Shift", "A"}));
+  m_move_buttons->SetControlExpression(MoveButtons::Right, hotkey_string({"Shift", "D"}));
+  m_move_buttons->SetControlExpression(MoveButtons::Forward, hotkey_string({"Shift", "W"}));
+  m_move_buttons->SetControlExpression(MoveButtons::Backward, hotkey_string({"Shift", "S"}));
+
+  m_speed_buttons->SetControlExpression(SpeedButtons::Decrease, hotkey_string({"Shift", "`1`"}));
+  m_speed_buttons->SetControlExpression(SpeedButtons::Increase, hotkey_string({"Shift", "`2`"}));
+  m_speed_buttons->SetControlExpression(SpeedButtons::Reset, hotkey_string({"Shift", "F"}));
+
+  m_other_buttons->SetControlExpression(OtherButtons::ResetView, hotkey_string({"Shift", "R"}));
+
+  m_fov_buttons->SetControlExpression(FieldOfViewButtons::IncreaseX,
+                                      hotkey_string({"Shift", "`Axis Z+`"}));
+  m_fov_buttons->SetControlExpression(FieldOfViewButtons::DecreaseX,
+                                      hotkey_string({"Shift", "`Axis Z-`"}));
+  m_fov_buttons->SetControlExpression(FieldOfViewButtons::IncreaseY,
+                                      hotkey_string({"Shift", "`Axis Z+`"}));
+  m_fov_buttons->SetControlExpression(FieldOfViewButtons::DecreaseY,
+                                      hotkey_string({"Shift", "`Axis Z-`"}));
+}
+
+ControllerEmu::ControlGroup* FreeLookController::GetGroup(FreeLookGroup group) const
+{
+  switch (group)
+  {
+  case FreeLookGroup::Move:
+    return m_move_buttons;
+  case FreeLookGroup::Speed:
+    return m_speed_buttons;
+  case FreeLookGroup::FieldOfView:
+    return m_fov_buttons;
+  case FreeLookGroup::Other:
+    return m_other_buttons;
+  default:
+    return nullptr;
+  }
+}
+
+void FreeLookController::Update()
+{
+  g_freelook_camera.SetControlType(FreeLook::GetActiveConfig().camera_config.control_type);
```

Comment:
Is this really necessary on `Update` (which I assume is either on every frame or every poll interval for controllers - so a LOT of times)? It feels like this could be done once during startup and then when configuration changes.

I mean, it _does_ have a check if the change is even necessary; but it still feels weird to be here.

Reply:
This was left here because the control type did effect whether the camera was active.  Additionally, in multiview each controller was tied to an individual camera.

Considering multiview is not a thing _yet_, I'll move this back to `RenderBase` until that PR.